docs/_data/meetups/usa-portland.yaml

```diff
@@ -0,0 +1,6 @@
+region: North America
+country: USA
+state: Oregon
+city: Portland
+notes: |
+  Organised by [@WriteTheDocsPDX](https://twitter.com/WriteTheDocsPDX)
```

Comment:
This seems broken because there is no meetup website. We should raise an error in the extension here, I think?